docs/apache-airflow/tutorial.rst

```diff
@@ -184,14 +184,21 @@ Adding DAG and Tasks documentation
 ----------------------------------
 We can add documentation for DAG or each single task. DAG documentation only support
 markdown so far and task documentation support plain text, markdown, reStructuredText,
-json, yaml.
+json, yaml. The DAG documentation must be written in a doc string at the beggining of the DAG file to be taken
+in account. Below you can find some examples on how to implement task and DAG docs, as well as screenshots:
```

Comment:
> The DAG documentation must be written in a doc string at the beggining of the DAG file to be taken
in account

This isn't quite true.  It just has to be markdown.  It only has to be a module docstring if you want to create it by referencing the variable `__doc__`.  

You could do something like this anywhere in the file:

```
dag.doc_md = """
# Hello
"""
```

But it is true that using doc_md.__doc__ is a convenient and elegant way to add dag documentation.

Reply:
Great. Thanks for pointing that out. I'll add this to the documentation as well